### ISSUE TITLE
Error trying to start business network. Error: No valid responses from any peers.

### ISSUE THREAD

**Issue by Cielo2017** [hyperledger/composer#4634]
After I created a program and finished the model design/ query.qry / acl. I tested it at local playground and it ran successfully. When I tried to create the admin card (run composer network start), I failed. It launched a ccenv peer without response.

## Context

I couldn't find any other similar mistake in Google. It confuses me.

These are the error messages.

cielo@cielo-ThinkPad-E550:~/cprogrames/cmusic/dist$ composer network start -n cmusic -c PeerAdmin@hlfv1 -V twork start -n cmusic -c PeerAdmin@hlfv1 -V start -n cmusic -cadmin
Starting business network cmusic at version 0.0.1

Processing these Network Admins: 
userName: admin

✖ Starting business network definition. This may take a minute...
Error: Error trying to start business network. Error: No valid responses from any peers.
Response from attempted peer comms was an error: Error: REQUEST_TIMEOUT
Command failed

cielo@cielo-ThinkPad-E550:~/cprogrames/cmusic/dist$ composer network start -n cmusic -c PeerAdmin@hlfv1 -V twork start -n cmusic -c PeerAdmin@hlfv1 -V start -n cmusic -cadmin
Starting business network cmusic at version 0.0.1

Processing these Network Admins: 
userName: admin

✖ Starting business network definition. This may take a minute...
Error: Error trying to start business network. Error: No valid responses from any peers.
Response from attempted peer comms was an error: Error: failed to execute transaction f3d0e4a4cfa31b72766e4to execute transaction f3d0e4a4cfa31b72766e4ecute transaction f48e83o register cmusic:0.0.1 haincode cmusic3d0e4a4cfa31b72766e4a4cfa31b72766e475ec4b1ab48e83o register cmuhainc
Command failed

## Expected Behavior

 composer network start command should run successfully.

# Actual Behavior

Failed to create card. And launched a weird peer with a random name.
When I try to run again:composer network start -n cmusic -c PeerAdmin@hlfv1 -V 0.0.1 -A admin -S adminpw
It said that it's already been launched. 

## Possible Fix

I don't know why and how to fix it.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)
There are some familiar issues but no one like mine.

## Context

I can't create the admin card which make me stop my programming.

## Your Environment

* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

my composer is v0.20.8,my Docker version is 18.09.4, build d14af54
node -v8.15.1
npm -v6.4.1

these are my peers:
f818aed96671 hyperledger/fabric-peer:1.2.1 "peer node start" 2 days ago Up 2 days 0.0.0.0:7051->7051/tcp, 0.0.0.0:7053->7053/tcp peer0.org1.example.com
57989248aadd hyperledger/fabric-ca:1.2.1 "sh -c 'fabric-ca-se…" 2 days ago Up 2 days 0.0.0.0:7054->7054/tcp ca.org1.example.com
447556b93e71 hyperledger/fabric-couchdb:0.4.10 "tini -- /docker-ent…" 2 days ago Up 2 days 4369/tcp, 9100/tcp, 0.0.0.0:5984->5984/tcp couchdb
45b9da98c156 hyperledger/fabric-orderer:1.2.1 "orderer" 2 days ago Up 2 days 0.0.0.0:7050->7050/tcp orderer.example.com

**Comment by Cielo2017** [comment]
The first container is the confusing peer.
Shouldn't I start a container whose name start with dev?
![Screenshot from 2019-04-20 11-11-32](https://user-images.githubusercontent.com/26533524/56453015-241d5d00-636d-11e9-888f-758428057004.png)

**Comment by Cielo2017** [comment]
the command I use is:composer network start -n cmusic -c PeerAdmin@hlfv1 -V 0.0.1 -A admin -S adminpw

I don't know why it is different after I copy that from bash. Sorry.

**Comment by NasimShourav** [comment]
Hello! 

I got the following error while starting the composer

composer network start -c PeerAdmin@hlfv1 -n test-bna -V 0.0.1 -A admin -S adminpw
Starting business network test-bna at version 0.0.1

Processing these Network Admins:
userName: admin

✖ Starting business network definition. This may take a minute...
Error: Error trying to start business network. Error: No valid responses from any peers.
Response from attempted peer comms was an error: Error: failed to execute transaction 00d2f68d865e9906208d0ce8c7273a4250801d6a3e113d4ddeef1b378c8bf32f: error starting container: error starting container: Failed to generate platform-specific docker build: Error returned from build: 1 "npm ERR! code EAI_AGAIN
npm ERR! errno EAI_AGAIN
npm ERR! request to https://registry.npmjs.org/composer-common failed, reason: getaddrinfo EAI_AGAIN registry.npmjs.org:443

npm ERR! A complete log of this run can be found in:
npm ERR! /root/.npm/_logs/2019-05-19T16_31_01_444Z-debug.log
"
Command failed

I heard that this happen if someone is behind proxy. I changed the DNS to google Public DNS (For IPv4: 8.8.8.8 and/or 8.8.4.4) but still getting the same error.
